pulp_ansible/app/tasks/collections.py

```diff
@@ -203,6 +226,7 @@ def create_collection_from_importer(importer_result):
         collection_version = CollectionVersion(
             collection=collection,
             **collection_info,
+            requires_ansible=importer_result.get("requires_ansible"),
```

Comment:
`None` for now, but it will get values once `galaxy-importer` is released

Reply:
So `galaxy-importer` will carry the implementation and pulp_ansible will return `None` until `galaxy-importer` releases it? I'm ok with that, but I wanted to confirm my understanding.

Reply:
yes that is correct - we have merged in the galaxy-importer implementation here: https://github.com/ansible/galaxy-importer/pull/110